src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -297,7 +326,7 @@ public FeedbackSessionQuestionsBundle getFeedbackSessionQuestionsForInstructor(
 
         if (fsa == null) {
             throw new EntityDoesNotExistException(
-                    "Trying to get a feedback session that does not exist.");
+                    ERROR_GET_NON_EXISTENT_SESSION);
```

Comment:
This too. Do also combine the other thrown exceptions in the rest of the changed code below onto one line where applicable (if characters do not exceed the 110 char limit)